src/norm/sqlite.nim

```diff
@@ -414,3 +443,35 @@ proc selectManyToMany*[M1: Model, J: Model, M2: Model](dbConn; queryStartEntry:
   const fkColumnFromJoinToManyStart: string = J.getRelatedFieldNameTo(M1)
   const fkColumnFromJoinToManyEnd: string = J.getRelatedFieldNameTo(M2)
   selectManyToMany(dbConn, queryStartEntry, joinModelEntries, queryEndEntries, fkColumnFromJoinToManyStart, fkColumnFromJoinToManyEnd)
+
```

Comment:
This is the second of 2 key procs for this PR!
This allows you to query the many-to-many relationships of multiple entries at once!